eli5/xgboost.py

```diff
@@ -91,7 +93,9 @@ def explain_prediction_xgboost(
         target_names=None,
         targets=None,
         feature_names=None,
-        vectorized=False):
+        vectorized=False,
+        expand_missing_features=False,
```

Comment:
Right, that's a valid point! Let me work with it a little more, I also have a fairly limited experience with it :)